### ISSUE TITLE
Visible pop at the end of some out animations, e.g. outElastic

### ISSUE THREAD

**Issue by mariusk** [Nazariglez/pixi-tween#9]
For some of the "out" style animations (outElastic) there is very often a visual pop, where the last time step snaps everything into place. This piece of code is probably to blame:

```javascript
      let t = this._elapsedTime+deltaMS;
      let ended = (t>=time);

      this._elapsedTime = ended ? time : t;
      this._apply(time);
```

Together with the visual pop I'm seeing, I'm guessing the code is somewhat incorrect. I'm not sure what the deltaMS value is (is it the next natural timestep, or is it time elapsed), but I'll speculate that it is the next natural timestep. If so, we shouldn't immediately set time completed at THIS time, but instead apply just normal (non-ended time) and modify the next timestep so it roughly matches the actual time left.

Instead of digging into the details, I did find a simple workaround that removes the visual pop however, that others also might want to do if the experience the pop. Simply replace this line:

```javascript
      this._elapsedTime = t;
```

Basically we disregard the actual end time and just run with whatever time will elapse. I'm not sure why this seems to remove the pop and I haven't dug further into fixing the underlying cause, but at least everything is smooth, worst case with less precision on time elapsed.
